src/blog/2022/11/flowforge-1-1-released.md

```diff
@@ -0,0 +1,62 @@
+---
+title: FlowForge 1.1 released
+subtitle: Node-RED editor on Devices, persistant file storage and import snapshots from outside FlowForge
+description: Node-RED editor on Devices, persistant file storage and import snapshots from outside FlowForge
```

Comment:
Not sure when the subtitle or description is used, but it seems they shouldn't be the same.

Reply:
One is used on the blog index, the other in the header of each blog post. I think it's appropriate for them to be the same for our usage.